pkg/loader/translate.go

```diff
@@ -84,6 +86,17 @@ func columnToArg(mysqlType string, c *pb.Column) (arg interface{}) {
 			var str string = string(c.GetBytesValue())
 			return str
 		}
+		// issues: TOOL-1346 and TOOL-1374
+		// for downstream = mysql
+		// it work for tidb to update binary by translating []byte to uint64
+		if mysqlType == "bit" {
+			val, err := types.BinaryLiteral(c.GetBytesValue()).ToInt(nil)
```

Comment:
change `columnToArg` to return (arg interface{}, err error)
can't just return nil, then the caller can't handle it